recipes/vtkplotter/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "vtkplotter" %}
+{% set version = "2020.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: d9db1c6f1e267ef672cf7721cdc270057d8e22428b8aa04e1e0a2b119b1d28e9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
can this be noarch python?

Reply:
Yup, source and data only so I guess that is a "yes". Implemented and pushed